ip/draft-ietf-detnet-ip.xml

```diff
@@ -572,79 +571,48 @@ DetNet                    |L2/SbN|          |L2/SbN|
-        </t>
-        <t>
-          DetNet flows forwarded via IP have more limited aggregation
-          options, due to the available traffic flow identification fields of
-          the IP solution.
-
-          One available approach is to manage the resources
-          associated with a DSCP identified traffic class and to map (remark)
-          individually controlled DetNet flows onto that traffic class.  This
-          approach also requires that nodes support aggregation ensure that
-          traffic from aggregated flows are placed (shaped/policed/enqueued)
-          in a fashion that ensures the required DetNet service is preserved.
+     <section title="DetNet Flow Aggregation" anchor="aggregation">
+       <t>
+         As described in <xref
+         target="I-D.ietf-detnet-data-plane-framework"/>, the ability to
+         aggregate individual flows, and their associated resource
+         control, into a larger aggregate is an important technique for
+         improving scaling by reducing the state per hop.  DetNet IP
+         data plane aggregation can take place within a single node,
+         when that node maintains state about both the aggregated and
+         component flows.  It can also take place between nodes, where
+         one node maintains state about only flow aggregates while the
+         other node maintains state on all or a portion of the component
+         flows.  In either case, the management or control function that
+         provisions the aggregate flows must ensure that adequate
+         resources are allocated and configured to provide combined
+         service requirements of the component flows.  As DetNet is
+         concerned about latency and jitter, more than just bandwidth
+         needs to be considered.
         </t>
         <t>
-          Additional details of the traffic control capabilities needed at 
-          a DetNet-aware node may be covered in separate future
-          documents.  Management and control plane mechanisms will also need
-          to ensure that the service required on the aggregate flow 
-          are provided, which may include the  discarding or remarking
-          mentioned in the previous sections.
-        </t>
-      </section>
+          From a single node perspective, the aggregation of IP flows
+          impacts DetNet IP data plane flow identification and resource
+          allocation.  As discussed above, IP flow identification uses
+          the IP "6-tuple" for flow identification.  DetNet IP flows can
+          be aggregated using any of the 6-tuple fields defined in <xref
+          target="ip-flow-id"/>.  The use of prefixes, wildcards,
+          bitmasks, and value ranges allows a DetNet node to identify
+          aggregate DetNet flows.  From a resource allocation
+          perspective, DetNet nodes must provide service to a aggregate
+          and not on a component flow basis.
```

Comment:
Terminology: individual flows vs, component flows? Do we need a new term (component flow)?